src/content/en/tools/chrome-user-experience-report/index.md

```diff
@@ -90,6 +90,36 @@ Defined by the [HTML specification](https://html.spec.whatwg.org/#event-load):
 finished loading.” - 
 [MDN](https://developer.mozilla.org/en-US/docs/Web/Events/load).
 
+#### First Input Delay {: #first-input-delay }
```

Comment:
@rviscomi to stay consistent, could we add the the "Defined by <spec>" preambles for all the new metrics? It's useful to have a direct reference to the spec that spells out the metric.